composer.json

```diff
@@ -29,6 +29,7 @@
         "sonata-project/core-bundle": "~2.3,>=2.3.1"
     },
     "require-dev": {
+        "rhumsaa/uuid": "^2.8",
```

Comment:
The UuidType and Uuid object to generate a uuid and have it set in the entity so doctrine knows what to convert. I will try to implement the extra mock objects and remove the uuid package dependency